django/db/migrations/writer.py

```diff
@@ -509,7 +513,8 @@ def serialize(cls, value):
 %(imports)s
 
 class Migration(migrations.Migration):
-%(replaces_str)s
+%(replaces_str)s%(initial_str)s
+
```

Comment:
extra newline if initial_str is empty -- I think iniitial_str should have a \n at the end like replaces_str does